taxonomies/data_quality.txt

```diff
@@ -656,16 +682,20 @@ description:en: Sodium prepared value is over 1000g per 100g\,which is impossibl
 
 < en:Nutrition errors
 en: Nutrition value over 1000 - Starch
-description:en: Starch value is over 1000g per 100g\,which is impossible.
+da: Næringsmængde over 1000 - Stivelse
+<<<<<<< HEAD
```

Comment:
There's a merge issue here

Reply:
Ah! I thought I’d gotten all of them. Oops.